### DISCUSSION TITLE
Documentation on DB spec (Not a Bug, requesting Documentation)

### DISCUSSION THREAD

**Discussion by tn185075** [ory/hydra#3116] [Q&A] [answered]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/hydra/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/hydra/blob/master/CONTRIBUTING.md).
- [ ] This issue affects my [Ory Cloud](https://www.ory.sh/) project.
- [ ] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [ ] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe the bug

Cannot find any documentation on DB spec, how data is encrypted and stored? What exactly each column would mean in every table?

### Reproducing the bug

Try to find related doc on Docs site 😃

### Version

v1.10.7

**Comment by aeneasr** [comment, marked as answer]
Hey there, sorry for the late reply on this. We do not have documentation on these topics. Unfortunately it will require to read some of the code. What are you trying to achieve?

**Reply by tn185075** [reply]
I wanted to make use of access token information stored by hydra in postgres (in refresh token flow).
